04.dedicated-server/01.getting-started/docs.md

```diff
@@ -79,24 +103,82 @@ Note: If you run multiple servers on the same host, port numbers are automatical
 
 >>>>> You can test if your network configuration is correct using: <http://www.yougetsignal.com/tools/open-ports/>
 
+## Adding maps
+
+To add maps to your server, first put the `Map.Gbx` files somewhere in the `UserData/Maps/` folder. Then, for each map you want to add to the playlist, add the following line to your match settings file:
+
+```
+<map><file>Downloaded\FilenameGoesHere.Map.Gbx</file></map>
+```
+
+These should be added just before the `</playlist>` ending tag. It might look something like this:
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<playlist>
+  <gameinfos>
+    <!-- ... -->
+  </gameinfos>
+
+  <filter>
+    <!-- ... -->
+  </filter>
+
+  <script_settings>
+    <!-- ... -->
+  </script_settings>
+
+  <startindex>0</startindex>
+  <map><file>Downloaded\My cool track.Map.Gbx</file></map>
+  <map><file>Downloaded\My other cool track.Map.Gbx</file></map>
+  <map><file>My Maps\Going_in_circles.Map.Gbx</file></map>
+</playlist>
+```
+
+If you choose to install a server controller, it is most likely capable of automatically adding maps to your matchsettings file. You do however need an initial matchsettings file, so it's good to know how to do it manually.
 
 ## Starting the server
 
-The minimal command line to start the server is `ManiaPlanetServer /nodaemon /Title=TitleId@CreatorLogin /dedicated_cfg=DedicatedCfgFile /game_settings=MatchSettingsFile`.
+The minimal command line to start the server is `ManiaPlanetServer /nodaemon /dedicated_cfg=DedicatedCfgFile /game_settings=MatchSettingsFile`.
 
->>>>> Always use the `/nodaemon` option to be able to see the potential errors.
+>>>>> Always use the `/nodaemon` option to be able to see any potential errors.
 
-*[Complete list of ManiaPlanetServer arguments](../references/command-line)*
+Here's a full list of options you can run the server with:
 
-Please note that without options, the server will be executed in the background which make it difficult to see errors, add the `/nodaemon` option to make it execute in the foreground.
+|  Option  |  Description  |
```

Comment:
I would not put all options here as we have a dedicated page. It's a quick start so I would just not mention them here.

Reply:
Ah, makes sense, I thought that link gave me a 404 before so I just re-wrote the entire table. Will add the missing options.